docs/_indicators/AtrStop.md

```diff
@@ -76,6 +76,22 @@ IReadOnlyList<AtrStopResult>
 
 See [Utilities and helpers]({{site.baseurl}}/utilities#utilities-for-indicator-results) for more information.
 
+## Streaming
```

Comment:
Include documentation for both buffer and stream style implementations here.  If we're missing one of those implementations, add the implementation and the documentation.  I see this on several files and places in this PR.